scripts/scil_run_commit.py

```diff
@@ -107,6 +106,14 @@ def _build_arg_parser():
                         'If not set, uses a binary mask computed from '
                         'the streamlines.')
 
+    g0 = p.add_argument_group(title='COMMIT2 options')
+    g0.add_argument('--commit2', action='store_true',
+                    help='Run commit2, requires .h5 as input and will force\n'
+                         'ball&stick model.')
+    g0.add_argument('--lambdas', type=float, default=1e-4,
+                    help='Run commit2, requires .h5 as input and will force\n'
+                         'ball&stick model [%(default)s].')
+
```

Comment:
Can we name the variable --lambda_commit2 ou something similar?

And I think we should add the --lambda_commit1

Also, please adjust the help as right now it is a copy paste of the above line